contracts/tests/clarigen-types.ts

```diff
@@ -1330,7 +1330,7 @@ export const contracts = {
         value: 4n,
       },
       tokenDecimals: 8n,
-      tokenName: "sBTC Mini",
+      tokenName: "sBTC",
```

Comment:
Good catch

Reply:
Yeah, this was autogenerated by "make build" after I had a CI failure caused by this 😅